src/app/app.component.ts

```diff
@@ -17,8 +18,13 @@ export class AppComponent {
   photoURL: string;
 
   constructor(public authService: AuthService,
-              private router: Router) {
+              private router: Router,
+              private titleService: Title) {
     this.title = environment.title;
+    
+    // Issue #3 Resolved: Set Title using environment.title
```

Comment:
Please delete this comment. Line above doesn't require comment to understand what is happening here. Resolved issue is captured in your PR. We don't want useless comments. Code should be self-documented. Have a look at "clean code" practice.

Reply:
OK, sure.